docs/installation/proxmox.mdx

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 6
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Screenshot from "../../src/components/Screenshot";
+
+# Proxmox
```

Comment:
Bei der Proxmox Anleitung ist mir der Nutzen noch nicht ganz klar. Ich bin allerdings auch kein Proxmox Nutzer und weiß daher nicht was üblich ist. Hast du ein, zwei Links wie andere das machen? Konkret wird hier ja "nur" der Image Typ und die Speicheranforderungen erwähnt.

Reply:
Ja könnte man im Prinzip auch weglassen. Obwohl es sicher hilft wenn man z.b. Proxmox nür für seine HA Installation nutzt und sich damit noch nicht so gut damit auskennt. Es gibt einige HA user, die wie ich HA, auf Proxmox installieren. Das geht recht einfach da es Installations Scripts gibt. Und ev. möchten diese User evcc als Linux VM einrichten um den Konfigurations Assistenten zu nutzen (so wie ich). Dazu kommt man mit dieser Anleitung innerhalb 10 min zum Ziel. Daher macht es schon auch Sinn.

Reply:
Wenn man vorhat evcc auf einem Linux Home Server, zu betreiben, kann Proxmox auch eine flexible Alternative darstellen, die es ermöglicht neben evcc auch noch andere systeme zu installieren. Eine evcc Installation auf [Proxmox] (https://www.proxmox.com/en/) verhält sich
danach genau so wie eine Installation auf einem dedizierten Linux server.

Reply:
Hattest du das hier https://tteck.github.io/Proxmox/#evcc-lxc gestern in #general bei Slack gesehen? Macht es Sinn das auch hier zu verlinken?

https://tteck.github.io/Proxmox/#evcc-lxc

Reply:
> Hattest du das hier https://tteck.github.io/Proxmox/#evcc-lxc gestern in #general bei Slack gesehen? Macht es Sinn das auch hier zu verlinken?

unbedingt!